File: ruby/f00.inheritance/Student.rb
#!/usr/bin/ruby
# class declaration
class Student < Person
  ###### Mutators ######
  def setGPA (gpa=0.0)
    @gpa = gpa         # set instance data
  end
  
  ###### Accessors ######
  def getGPA
    @gpa               # return instance data
  end
end

File: python/f00.inheritance/Student.py
#!/usr/bin/python
from Person import Person    # include Person.py

# class declaration
class Student(Person):
  ###### Mutators ######
  def setGPA (self,gpa):
    self.gpa = gpa      # set data

  ###### Accessors ######
  def getGPA(self):
    return self.gpa      # return data
